Update/some_008.txt

```diff
@@ -1393,7 +1393,7 @@ void main()
 //r見るとその人は、介護士さんだった。あまり親しくなかったので名前は知らないけど、時々シフトが一緒になって、挨拶くらいは交わして顔は覚えている。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "見るとその人は、介護士さんだった。あまり親しくなかったので名前は知らないけど、時々シフトが一緒になって、挨拶くらいは交わして顔は覚えている。",
-			NULL, "I recognized her, she was one of the caregivers. I didn't know her name because we were never very close, but we occasionally had shifts together. I recalled exchanging greetings with her face to face.", Line_Normal);
+			NULL, "It was one of the caregivers. I didn't know her name because we were never very close, but we occasionally had shifts together. At the very least, I knew her enough to recognize her face and say hi to her.", Line_Normal);
```

Comment:
```suggestion
			NULL, "It was one of the caregivers. I didn't know her name because we were never very close, but we occasionally had shifts together. At the very least, I knew her well enough to say hi to her when I recognized her face.", Line_Normal);
```